packages/azure-http-specs/specs/service/multi-service/client.tsp

```diff
@@ -0,0 +1,55 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/spector";
+import "@typespec/http";
+import "@typespec/versioning";
+
+using Azure.ClientGenerator.Core;
+using Spector;
+using Http;
+using Versioning;
+
+@scenarioDoc("""
+  Test that a client can expose operations from multiple services.
+  This scenario demonstrates creating a unified client that provides access to 
+  operations from different services using the @client decorator.
+  
+  ```ts
+  const client = new MultiServiceClient();
+  
+  // Call operations from first service - uses api-version 1.1.0
+  client.firstOne();
+  client.firstTwo();
+  
+  // Call operations from second service - uses api-version 2.1.0
+  client.secondOne();
+  client.secondTwo();
+  ```
+  """)
+@scenario
+@route("/multi-service")
+namespace Service.MultiService.Scenario {
+  @get
+  @route("/first/one")
```

Comment:
I assume we should not write these REST decorators in client.tsp

They belong to the definition of the REST in main.tsp
If we are duplicating all of them here, something likely not right.

---

Also, for the op in client.tsp, e.g. this `firstOne`, it also not referring to the op in main.tsp (not using `firstOne is Service.MultiService.FirstService.Operations.firstOne`

---

Overall, this test kind of just define new op in client.tsp